src/WebJobs.Script/Description/Rpc/WorkerLanguageInvoker.cs

```diff
@@ -75,7 +75,7 @@ protected override async Task<object> InvokeCore(object[] parameters, FunctionIn
 
             ScriptInvocationResult result;
             _logger.LogDebug($"Sending invocation id:{invocationId}");
-            await _functionDispatcher.InvokeAsync(invocationContext);
+            _functionDispatcher.Invoke(invocationContext);
```

Comment:
Part of revert

Reply:
we would still need this to look up a channel that started in placeholder mode and did not complete initialization before specialization is called

Reply:
this code does do that - an instance of "LanguageWorkerChannel" is registered as soon as it is created. The difference is that the awaiting for process initialization is done when we call "function load", and not when we call for a reference to the "LanguageWorkerChannel" class